neuronsimulator/forms.py

```diff
@@ -77,17 +75,20 @@ def clean_resp_neu_ids(self):
         but require at least one resp neuron at this stage of test
         will deal with this case later: resp_neu_ids=None and use a threshold value
         """
-        if resp_neu_ids is None:
+        if len(resp_neu_ids) == 0:
             raise forms.ValidationError(
-                "need to select at least one response neuron (for testing!)"
+                "You must select at least one response neuron (for testing)."
             )
         else:
             resp_neu_arr = resp_neu_ids.split(",")
+            invalid_neurons = []
             for resp_neuron in resp_neu_arr:
                 if resp_neuron not in neurons:
-                    raise forms.ValidationError(
-                        "found invalid neuron in response neuron list, please re-select!"
-                    )
+                    invalid_neurons.append(resp_neuron)
```

Comment:
Just a couple of [options that are a bit more "pythonic"](https://www.geeksforgeeks.org/python-difference-two-lists/).

Use set difference operation:
```python
invalid_neurons = list( set(resp_neu_arr) - set(neurons) )
```

Using list comprehension:
```python
invalid_neurons = [i for i in resp_neu_arr if i not in neurons]
```

Reply:
@lparsons Nice suggestions., thanks. I tested both in Django shell and opt to use 2nd one. Just pushed another commit.